### DISCUSSION TITLE
Selecting tables with same name

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#5170] [Ideas]
I have several mySQL DBs to manage, with lots of duplicate table names. (Sort of a multi-tenancy implementation)
Consequently, when I try to select a table in the data provider, I always have a hard time choosing the right table to use.
It would be very great if the DB name is shown when selecting a table.

Suggestions #1)

**DB_1** > table1
**DB_2** > table1
**DB_1** > table2
**DB_2** > table2

Suggestions #2)

**Tables in DB_1**
table1
table2

**Tables in DB_2**
table1
table2

**Comment by mjashanks** [comment]
Thanks @dannyshin76  - agreed, I've hit this one too!
